utopia_tree.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the utopianTree function below.
+def utopianTree(n): 
+
+    #my code starts here      
+    number_of_periods = n
```

Comment:
Unecessary either keep one name or the other

Reply:
Fixed